CHANGELOG.md

```diff
@@ -11,6 +11,17 @@
 ### Terraform Runner
 - no changes
 
+## v1.7.3
+
+### CLI
+- New command `unipipe git` runs Git pull/push commands resiliently. It takes care of retrying and rebasing if needed to make sure a push will be successful.
```

Comment:
f: We add new changelogs always to the vNext section as multiple changes might be included in the next release. The new release number will then be picked, when we build the new release.